app/helpers/facility_reservations_helper.rb

```diff
@@ -4,8 +4,13 @@ def offline_category_collection
     I18n.t("offline_reservations.categories").invert.to_a
   end
 
-  def reservation_category_label(category)
-    I18n.t(category.presence, scope: "offline_reservations.categories", default: "")
+  def admin_category_collection
+    I18n.t("reservations.categories").invert.to_a
+  end
+
+  def reservation_category_label(reservation)
+    scope = reservation.is_a?(OfflineReservation) ? "offline_reservations.categories" : "reservations.categories"
```

Comment:
`class.name` in this case will return Reservation I believe - I was having this issue when I moved the categories constant to AdminReservation, the validation for the absence of a category in ReservationValidation was failing because it claimed categories wasn't defined on the class.